test/src/org/labkey/test/tests/targetedms/TargetedMSQCTest.java

```diff
@@ -860,6 +860,34 @@ public void testQCPlotExclusions()
         assertEquals("Unexpected number of included data points in plot SVG", 4, excludedPointCount);
     }
 
+    /*
+       Test coverage : Issue 45440: Moving range plot broken when choosing Percent of Mean or Standard Deviation y-axis options
+     */
+    @Test
+    public void testYAxisOptionsMovingRangePlot()
+    {
+        goToProjectHome();
+        PanoramaDashboard qcDashboard = new PanoramaDashboard(this);
+        QCPlotsWebPart qcPlotsWebPart = qcDashboard.getQcPlotsWebPart();
+        log("Enabling Moving range along with Levey-Jennings");
+        qcPlotsWebPart.checkPlotType(MovingRange, true);
+        qcPlotsWebPart.waitForPlots(PRECURSORS.length * 2, true);
+
+        log("Verifying standard deviations plots");
+        qcPlotsWebPart.setScale(QCPlotsWebPart.Scale.STANDARD_DEVIATIONS);
+        String svgPlotText = qcPlotsWebPart.getSVGPlotText("tiledPlotPanel-2-precursorPlot0");
+        assertFalse("Plot with standard deviations option is blank", svgPlotText.isEmpty());
+        log("SVG content SD" + svgPlotText);
```

Comment:
Do we always want to log this? Or should it be moved to only log when there's a failure?

Reply:
Removed